src/eq_schema/Question.test.js

```diff
@@ -101,6 +101,70 @@ describe("Question", () => {
         })
       );
     });
+
+    it("should create other answer if exists", () => {
+      const answers = [
+        {
+          type: "Checkbox",
+          id: "1",
+          options: [
+            {
+              id: "1",
+              label: "Option 1"
+            }
+          ],
+          other: {
+            option: {
+              id: "2",
+              label: "Other option"
+            },
+            answer: {
+              id: "2",
+              type: "TextField"
+            }
+          }
+        }
+      ];
+      const question = new Question(createQuestionJSON({ answers }));
+
+      expect(question.answers).toHaveLength(2);
+      expect(question.answers).toEqual(
```

Comment:
You can simplify this assertion. No need for `arrayContaining`, you can pass an array to `toEqual` itself. This also ensures that the order is correct, which `arrayContaining` does not.

```js
expect(question.answers).toEqual([
  expect.objectContaining({
    type: "Checkbox"
  }),
  expect.objectContaining({
    type: "TextField"
  })
]);
```